CRM/Contribute/Page/Tab.php

```diff
@@ -43,6 +43,13 @@ class CRM_Contribute_Page_Tab extends CRM_Core_Page {
   public $_contactId = NULL;
   public $_crid = NULL;
 
+  /**
+   * Array with statuses that mark a recurring contribution as inactive.
+   *
+   * @var array
+   */
+  private $inactiveStatuses = array('Cancelled', 'Chargeback', 'Refunded', 'Completed');
```

Comment:
This would be better in the ContributionRecur BAO - there *may* already be something that does this, otherwise would need to create a helper function.